src/cli/commands/init.js

```diff
@@ -100,11 +102,25 @@ export async function run(
     }
 
     let answer;
+    let validAnswer = false;
 
     if (yes) {
       answer = def;
-    } else {
-      answer = (await reporter.question(question)) || def;
+    } else { 
+      //loop until a valid answer is provided, if validation is on entry
+      if (entry.validation) {
+        while (!validAnswer) {
+          answer = (await reporter.question(question)) || def;
+          //validate answer
```

Comment:
Can you add a space after `//` to pad this out? Just to keep consistency with how we do comments elsewhere. eg:

```
// loop
```

rather than

```
//loop
```